libs/shared/ui/src/lib/components/badge/badge.tsx

```diff
@@ -0,0 +1,123 @@
+import { type VariantProps, cva } from 'class-variance-authority'
+import { type ComponentPropsWithoutRef, type ElementRef, forwardRef } from 'react'
+import { twMerge } from 'tailwind-merge'
+
+const badgeVariants = cva(['inline-flex', 'items-baseline', 'shrink-0', 'font-medium', 'rounded'], {
+  variants: {
+    color: {
+      neutral: [],
+      red: ['text-red-500'],
+      purple: ['text-purple-500'],
+      sky: ['text-sky-500'],
+      green: ['text-green-500'],
+      yellow: ['text-yellow-500'],
+      brand: ['text-brand-500'],
+    },
+    variant: {
+      outline: ['border'],
+      surface: ['border'],
```

Comment:
`outline` and `surface` are the same?

Reply:
Nope, the `compoundVariants` are different